### ISSUE TITLE
Using rows and cols

### ISSUE THREAD

**Issue by mats68** [ShakingMap/shaking-react-form#3]
Hi,
Did you think about how to organize fields in rows and columns, in a grid system ?
I didn't find a way to put several fields in a single row:

`<div class="row">
  .. field1
  ..field2
</div>
`

**Comment by zhaoyao91** [comment]
No, it does not support rows-cols. This package focuses on the functionality of generating form with validations, but rich layouts are not considered much, to reduce the complex.

**Comment by mats68** [comment]
Hey, your framework is pretty slick an not complex, and that's why i find it cool, but it's really important not to have every input in a whole row. Btw. i found it worked when i put the whole form in a row.

**Comment by zhaoyao91** [comment]
I will consider a solution in my new form project. thanks for this idea

**Comment by zhaoyao91** [comment]
as you can put a form in a col, how about offer a way to connect many form into a big one ？

**Comment by mats68** [comment]
Hi, Not sure what you meant in you last comment. But anyhow it seems to work like this (with materialize):
`return <div className="row">
      <form onSubmit={this.onFormSubmit.bind(this)}>`

Then i can add col's in my field-defs.
Anyhow i love this framework because it's so slick. It was very easy to put whole form readonly.
With old autoform had quite some bugs.

**Comment by zhaoyao91** [comment]
Yes, I've finish a mechanism to connect forms together in this manner in the new project

```
     <div>
        <div className="row">
            <div className="col col-xs-6">
                <Form {...{
                    ref: 'form1',
                    // ...
                }}/>
            </div>
            <div className="col col-xs-6">
                <Form {...{
                    ref: 'form2',
                    // ...
                }}/>
            </div>
        </div>
        <Form {...{
            subForms: ()=> {
                return {
                    form1: this.refs.form1,
                    form2: this.refs.form2
                }
            },
            // ...
        }}>
            <button className="btn btn-primary">Submit</button>
        </Form>
    </div>
```

the new project has finished development, I will release it soon for try and test after docs being written. As the new project has super set of functions over this one, and the apis and structures are better designed, I will suggest use the new project as the replacement of this one, and will not add new functions to this project, sorry. please follow the the new [rf-form](https://github.com/ShakingMap/rf-form)

**Comment by zhaoyao91** [comment]
hi, I have released the new project [rf-form](https://github.com/ShakingMap/rf-form). would you like to have a try and report bugs if found?
